cpp/src/io/avro/avro_gpu.cu

```diff
@@ -261,11 +261,7 @@ extern "C" __global__ void __launch_bounds__(NWARPS * 32, 2)
   } else {
```

Comment:
@rgsl888prabhu  Aren't lines 256-257 also cooperative loading?

Reply:
Ahh, missed that. Will push changes.